ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -505,6 +505,9 @@ then
 			set AUX_MODE none
 		fi
 		camera_trigger start
+
+		param set CAM_FBACK_MODE 1
```

Comment:
CAM_FBACK_MODE? In general should we push the param handling into the modules themselves?